### ISSUE TITLE
Идея: выводить город/страну пользователя рядом с его тайтлом.

### ISSUE THREAD

**Issue by ephimoff** [vas3k/vas3k.club#535]
Очень много постов и комментов релевантны только для определенных географических зон и видеть сразу откуда топиккастер/комментатор было бы отлично.

**Comment by devcooch** [comment]
Хм. С одной стороны, я понимаю твою логику, с другой стороны проверил Твиттер, Фейсбук, Хабр, vc.ru, Reddit - нигде такого нет, локация пользователя показывается только при открытии профиля.

**Comment by Galichev** [comment]
Мне кажется, и в случае публикации и в случае комментария это должно быть ясно из контекста. В статье будет явно написано, а комментарий из другого региона просто не будут оставлять. Можно несколько примеров где это будет полезно?

**Comment by ephimoff** [comment]
Да, я понимаю спорность этого. Но я наталкивался несколько раз на посты/комменты, где мне было бы удобно знать контекст (читай: страну) автора. Обычно это что-то связанное с работой (собеседования, советы, процессы) : например автор пишет "а вот у нас в компании есть такое...". И контекст тут помог бы. Потому что реалии в разных странах разные.
Другой пример: покупки. Например, кто-то рекомендуют бренд или товар и не очень понятно есть ли такой товар в моей стране. 
Так сходу конкретные посты не могу привести, посмотрел на первую страницу и мне кажется вот эти бы посты выиграли от наличия такой фичи:

- [Чем крута ваша компания](https://vas3k.club/question/7999/)
- [Что бы вы посоветовали человеку 20 лет, который делает первые шаги в вашей профессии?](https://vas3k.club/question/8006/)
- [Индексация з/п](https://vas3k.club/question/7956/)
